testing/python/amd/test_mfma_dtype_str_bug.py

```
@@ -0,0 +1,156 @@
+        ("bf16", T.bfloat16, T.float32),
+    ]:
+        try:
+            func = matmul_kernel(16, 16, 16, in_dt, out_dt)
+            mod = tilelang.compile(func)
+            print(f"  {name}: COMPILE OK")
+        except TypeError as e:
+            if "DataType" in str(e):
+                print(f"  {name}: BUG REPRODUCED - {e}")
+            else:
+                print(f"  {name}: UNEXPECTED ERROR - {e}")
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)